src/elements/icon/icon.jsx

```diff
@@ -21,7 +21,7 @@ const Icon = ({ src, alt, className, ...restProps }) => (
     role="img"
     {...restProps}
   >
-    <title>{alt}</title>
+    {alt && <title>{alt}</title>}
```

Comment:
We can keep this for the consistency but it should be changed:
- [ ] remove the title at all
- [ ] add `alt` as `aria-label` to the SVG
- [x] make it optional

`<title>` works as the `title` attribute, it shows a tooltip and this is not what we want. Alt is only for the accessibility when the user cannot see the image, but only can read the text; to my mind, `aria-label` fits it better.

Reply:
You confused me with the checked box. I thought you want to leave it as an optional and solve is later.

Reply:
Sorry, I just liked this type of list a bit more 🙂

Also, you can mark it If I am not mistaken.